lib/child-writer/renderer-utils.js

```diff
@@ -46,5 +46,5 @@ exports.getApplicationEntryPoint = () => {
   // On GNU/Linux, `pkexec` resolves relative paths
   // from `/root`, therefore we pass an absolute path,
   // in order to be on the safe side.
-  return path.join(CONSTANTS.PROJECT_ROOT, relativeEntryPoint)
+  return path.resolve(CONSTANTS.PROJECT_ROOT, relativeEntryPoint)
```

Comment:
Can you explain why this change is needed? Will it break other platforms? If not, should we send it as a different PR?